getting-started/getting-started.md

```diff
@@ -1,39 +1,53 @@
 ---
 description: >-
-  This quick-start guide covers configuring rewrite with Maven and Gradle and
-  using it to apply refactoring recipes.
+  This guide covers configuring rewrite with Maven or Gradle and using it to run refactoring recipes.
 ---
 
-# Quickstart: Maven and Gradle
+# Quick Start: Setting Up Your Project and Running Recipes
 
-Following this guide you will learn:
+To help orient you to OpenRewrite, let's walk through configuring a project to use the Maven or Gradle rewrite plugin. Then let's walk through running various types of recipes on said project and talk through the results.
 
-* How to apply the rewrite-maven-plugin to a Maven project
-* How to apply the rewrite-gradle-plugin to a Gradle project
-* How to configure, run, and inspect the results of built-in and external refactoring recipes
+In this guide you will:
+
+* [Clone a sample project](#step-1-clone-sample-project)
+* [Add the rewrite-maven-plugin or rewrite-gradle-plugin to your project](#step-2-add-rewrite-maven-plugin-or-rewrite-gradle-plugin-to-your-project)
+* [Activate a recipe so it can be run](#step-3-activate-a-recipe)
+* [Run a simple refactoring recipe](#step-4-run-a-simple-refactoring-recipe)
+* [Run a recipe with YAML configuration](#step-5-run-a-recipe-with-yaml-configuration)
+* [Add and run an externally created recipe](#step-6-running-recipes-from-external-modules)
+
+## Prerequisites
+
+This quick start guide assumes that you:
+
+* Are somewhat familiar with Java
+* Have worked with a Maven or a Gradle project before
+* Know how to run commands in the command line
+* Can run basic Git commands
 
 ## Step 1: Clone sample project
 
+The first step in this process is making sure you have some code to work with. We've created a [sample repository](https://github.com/openrewrite/spring-petclinic-migration) that you can use if you'd like. However, as all of the steps in this guide apply to any Java project built with Maven or Gradle, please feel free to use your own and skip to [Step 2](#step-2-add-rewrite-maven-plugin-or-rewrite-gradle-plugin-to-your-project).
```

Comment:
```suggestion
The first step in this process is making sure you have some code to work with. We've prepared a [sample repository](https://github.com/openrewrite/spring-petclinic-migration) that you can use if you'd like. However, as all of the steps in this guide apply to any Java project built with Maven or Gradle, please feel free to use your own and skip to [Step 2](#step-2-add-rewrite-maven-plugin-or-rewrite-gradle-plugin-to-your-project).
```
I'm changing the word "created" to "prepared", because spring-petclinic was created by the spring team as a demo spring app. We forked that and made it suitable for our purposes.